randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -299,7 +299,7 @@ Extra - room_id: [9]
           Any of the following:
```

Comment:
fix > Door to Data Hub too please.
![grafik](https://github.com/user-attachments/assets/225e93d5-2b42-4186-aaf0-5b81452d73e6)

Reply:
nvm im dumb